modules/core/src/main/java/org/apache/ignite/internal/processors/cluster/GridClusterStateProcessor.java

```diff
@@ -1867,11 +1984,83 @@ public BaselineAutoAdjustStatus baselineAutoAdjustStatus(){
     }
 
     /**
-     * @param activate Activate.
-     * @return Activate flag string.
+     * @param state Cluster state.
+     * @return Cluster state string representation.
+     */
+    private static String prettyStr(ClusterState state) {
+        switch (state) {
+            case ACTIVE:
+                return "activate cluster";
+
+            case INACTIVE:
+                return "deactivate cluster";
+
+            case ACTIVE_READ_ONLY:
+                return "activate cluster in read-only mode";
+
+            default:
+                throw new IllegalStateException("Unknown cluster state: " + state);
+        }
+    }
+
+    /**
+     * Checks that activation process is happening now.
+     *
+     * @param state Current cluster state.
+     * @param newState Cluster state after finish transition.
+     * @return {@code True} if activation process is happening now, and {@code False} otherwise.
+     */
+    private boolean activate(ClusterState state, ClusterState newState) {
+        assert state != null;
```

Comment:
GridArgumentCheck need to be here instead assert.

Reply:
Fixed